osvr/RenderKit/RenderManagerOpenGL.cpp

```diff
@@ -795,43 +841,19 @@ namespace renderkit {
         }
         checkForGLError("RenderManagerOpenGL::RenderDisplayInitialize end");
 
-		// Store the frame buffer that was active before we started rendering,
-		// so we can put it back when we finalize.
-		GLint fb;
-		glGetIntegerv(GL_FRAMEBUFFER_BINDING, &fb);
-		m_initialFrameBuffer = static_cast<GLuint>(fb);
-
```

Comment:
Aren't we worried that this will mess up client code that relies on the frame buffer being the way they left it?  At least one of these store/retrieve calls was put in there in response to an issue where a client was having problems.

Reply:
Ahh... looks like this as been handled in another section of the code below.

Reply:
Yes, there is now a boolean to control whether to save and restore state: m_storeClientGLState as this has a performance impact. By default state is saved and restored, but the ATW backend turns it off because the ATW thread has its own context, so it's safe to skip the save/restore steps.

Reply:
Excellent.